src/vmstats.app.src

```diff
@@ -1,14 +1,13 @@
 {application, vmstats, [
-  {description, "Tiny application to gather VM statistics for StatsD client"},
+  {description, "Tiny application to gather VM statistics"},
   {vsn, "1.0.1"},
   {registered, [vmstats_sup, vmstats_server]},
   {applications, [
     kernel,
-    stdlib,
-    statsderl
+    stdlib
   ]},
   {mod, {vmstats, []}},
-  {applications, [statsderl]},
+  {applications, []},
```

Comment:
The applications tuple should not be there twice. That stuff is apparently a bug. Let's remove this one altogether and keep the complete one above.